src/components/card/card.scss

```diff
@@ -1,26 +1,21 @@
 @import '../bootstrap/bootstrap';
-
```

Comment:
not a big deal and I would concede formatting should really be enforced through build step, but ideally I would like contributors to refrain from arbitrary formatting changes since it adds noise to a PR for no particular value. 

To that point, I opened #116 to make this happen.  👍

Reply:
Sorry about that. Opened the project in a foreign IDE and couldn't figure out how to disable the "remove line breaks on save" feature. I agree that this type of stuff isn't usually necessary.